src/ResourceManager/DeploymentManager/Commands.DeploymentManager/Az.DeploymentManager.psd1

```diff
@@ -0,0 +1,155 @@
+#
+# Module manifest for module 'AzureRM.DeploymentManager'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 8/25/2018
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '0.9.0'
```

Comment:
forgot one thing.. the version for netcore needs to be 0.2.0.

Example : https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Aks/Commands.Aks/Az.Aks.psd1